sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -116,6 +116,18 @@ static bool PrintPiTrace = false;
 // Sycl RT calls piTearDown().
 static sycl::detail::ESIMDEmuPluginOpaqueData *PiESimdDeviceAccess;
 
+// Single-entry cache for piPlatformsGet call.
+static pi_platform PiPlatformCache;
+static sycl::detail::SpinLock *PiPlatformCacheMutex =
```

Comment:
Please add a TODO comment that this is a memory leak

Reply:
Done

Reply:
Konst : Follow level_zero implementation

Reply:
Please, fix this as soon as possible.